ouroboros-consensus-test/test-consensus/Test/Consensus/Mempool.hs

```diff
@@ -17,7 +17,7 @@ import           Control.Monad.Except (Except, runExcept)
 import           Control.Monad.State (State, evalState, get, modify)
```

Comment:
This whole module's diff is just preserving the previous test logic. Hence we use `txForgetValidated` a lot.